yolo/data.yaml

```diff
@@ -0,0 +1,5 @@
+train: C:\Users\frede\Documents\Teamprojekt\teamproject-objectdetection\yolo\data\train\images # your path 
```

Comment:
those paths are absolute, this won't work for anyone else. Further it wont work for ppl using Unix as their pathing looks differently. 
You can utilize gatherFilePath() found in modules/moduleFileManagement.py which is returning a correct path, formatted for the underlying system (unix/win/...) to your supplied file.